test/utest/src/cuter_types_tests.erl

```diff
@@ -81,3 +81,92 @@ setup(Mod) ->
   {Mod, Attrs}.
 
 cleanup(_) -> ok.
+
+-spec convert_types_test() -> any().
+convert_types_test() ->
+  Modules = [examples_for_type_analysis, examples_for_type_analysis_pair],
+  Fn = fun(M) ->
+    {ok, AST} = cuter_cerl:get_core(M, false),
+    AST
+    end,
+  Xs = [{M, Fn(M)} || M <- Modules],
+  TagGen = fun() -> ok end,
+  Kmodules = [cuter_cerl:kmodule(M, AST, TagGen) || {M, AST} <- Xs],
+  Specs = cuter_types:specs_as_erl_types(Kmodules),
+  Expect = mfas_and_specs(),
+  As = lists:flatten([spec_assertions(E, Specs) || E <- Expect]),
+  ExpectMfas = lists:sort([Mfa || {Mfa, _Spec} <- Expect]),
+  GotMfas = lists:sort(dict:fetch_keys(Specs)),
+  [?assertEqual(ExpectMfas, GotMfas)] ++ As.
+
+
+mfas_and_specs() ->
+  [
+    {
+      {examples_for_type_analysis, id, 1},
+      [erl_types:t_fun([erl_types:t_any()], erl_types:t_any())]
+    },
+    {
+      {examples_for_type_analysis, inc, 1},
+      [erl_types:t_fun([erl_types:t_integer()], erl_types:t_integer())]
+    },
+    {
+      {examples_for_type_analysis, to_atom, 1},
+      [erl_types:t_fun([erl_types:t_sup(erl_types:t_integer(), erl_types:t_atom())], erl_types:t_atom())]
+    },
+    {
+      {examples_for_type_analysis, translate, 3},
+      [erl_types:t_fun(
+        [erl_types:t_tuple([erl_types:t_from_term(point), erl_types:t_number(), erl_types:t_number()]),
```

Comment:
All the `erl_types:t_from_term(Atom)` calls should be changed to `erl_types:t_atom(Atom)` calls, for simplicity.